content/06.administrative.md

```diff
@@ -0,0 +1,13 @@
+## Acknowledgments
+
+We thank the data generators and submitters of the Single-cell Pediatric Cancer Atlas.
+
+This work was funded through the Alex's Lemonade Stand Foundation Childhood Cancer Data Lab and Childhood Cancer Data Lab Postdoctoral Fellowship (SMF).
+
+## Author Contributions
+
+Author|Contributions| |---|---|{% for author in manubot.authors %} |{{author.name}}|{% if author.contributions is string %}{{author.contributions}}{% else %}{% for contribution in author.contributions %}{{ contribution }}{% if not loop.last %}, {% endif %}{% endfor %}{% endif %}|{% endfor %}
```

Comment:
```suggestion
| Author|Contributions| |---|---|{% for author in manubot.authors %} |{{author.name}}|{% if author.contributions is string %}{{author.contributions}}{% else %}{% for contribution in author.contributions %}{{ contribution }}{% if not loop.last %}, {% endif %}{% endfor %}{% endif %}|{% endfor %}
```
Something with the table isn't rendering right, but I think this will fix it?